core/config/docs/core.toml

```
@@ -303,6 +303,8 @@ DatabaseTimeout = '10s' # Default
 KeyBundleID = '7a5f66bbe6594259325bf2b4f5b1a9c900000000000000000000000000000000' # Example
 # CaptureEATelemetry toggles collecting extra information from External Adaptares
 CaptureEATelemetry = false # Default
+# CaptureAutomationCustomTelem toggles collecting automation specific telemetry
+CaptureAutomationCustomTelem = false # Default
```

It looks like we spell out `Telemetry` everywhere else in this file:
```suggestion
CaptureAutomationCustomTelemetry = false # Default
```
Is it important to say `Custom`? Or could that be left out?
```suggestion
CaptureAutomationTelemetry = false # Default
```